src/nanovg.h

```diff
@@ -124,6 +159,22 @@ void nvgCancelFrame(NVGcontext* ctx);
 void nvgEndFrame(NVGcontext* ctx);
 
 //
+// Composite operation
+//
+// Composite operation in NanoVG works between frames. The default composite
```

Comment:
Once you put the composite op as part of the state, no need to example the "between frames" thing.